nixos/modules/services/networking/multipath.nix

```diff
@@ -563,12 +563,9 @@ in {
     # with the `-1` argument which disables systemd calls. Invoke `multipath`
     # to display the multipath mappings in the output of `journalctl -b`.
     boot.initrd.kernelModules = [ "dm-multipath" "dm-service-time" ];
-    boot.initrd.secrets = {
-      "/etc/multipath/wwids" = "/etc/multipath/wwids";
```

Comment:
I'm concerned this may be necessary when mounting `/nix` over multipath? I definitely recall needing it at some point - would need to test this on our baremetal installations to know for sure.

Reply:
It might be, but see my comment here: https://github.com/NixOS/nixpkgs/pull/139833#discussion_r721621598